index.html

```diff
@@ -10,6 +10,7 @@
     <meta name="apple-mobile-web-app-title" content="Yugioh Calc"/>
     <meta name="format-detection" content="telephone=no"/>
     <meta name="theme-color" content="#ffffff"/>
+    <link rel="preload" href="node_modules/@fontsource/ubuntu-mono/files/ubuntu-mono-latin-400-normal.woff" as="font" type="font/woff2" crossorigin=""/>
```

Comment:
The file extension doesn't match the `type` attribute here.  Both should be `woff2` (we don't need `woff` because the browsers that support `<link rel="preload">` all support `woff2`)

Reply:
@jacksonrayhamilton Fixed in cea459b6db21673e9b76c7287c646257749ca4d2. Changing to `woff2` causes both preload tags to be the same, so I just consolidated down to a single preload tag.

Please resolve this conversation if this looks good to you now.